app/shop.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+from app.customer import Customer
+
+
+@dataclass
+class Shop:
+    name: str
+    location: list[int]
+    products: dict
+
+    def calculate_product_cost(self, customer: Customer) -> float:
+        total_price = sum(
+            quantity * self.products[product]
+            for product, quantity in customer.product_cart.items()
+        )
+
+        return total_price
+
+    def print_purchase_receipt(self, customer: Customer) -> None:
+        timestamp = 1609763621
+        actual_dt = (datetime.fromtimestamp(timestamp).
+                     strftime("%d/%m/%Y %H:%M:%S"))
```

Comment:
you can use simple `datetime.datetime.now().strftime('%d/%m/%Y %H:%M:%S')`

Reply:
not fixed

Reply:
No, I can`t use it. If I use it i`ll have 26 11 and in tests there is 04 01 and my tests will be failed

Reply:
![Screenshot from 2023-11-26 20-50-34](https://github.com/mate-academy/py-shop-trip/assets/122517203/c69dfc98-f0fa-4f59-a187-996842354f89)
Read checklist

Reply:
Oh, thanks, my mistake, will be more attentive in future